### PR TITLE
Updated to allow responsive webpage for both UID2 and EUID

### PR DESCRIPTION
**Pull request by tynewton**
Added a ton of CSS and some reorganizing of the handlebars template to allow both EUID and UID2 opt-out portals to be responsive.

### CODE REVIEW

**Comment by tynewton** [comment]
https://user-images.githubusercontent.com/35481925/193158889-14da35a5-905c-4a3a-9e0d-d1a02622fb01.mov

**Comment by tynewton** [comment]
Here's a screen recording of the initial page for reference.
https://user-images.githubusercontent.com/35481925/194675130-4c401995-da61-4ad7-8afd-0a207a8793b3.mov

File: public/locales_euid/en.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
 	"By entering your email below, you confirm that this is your information.": "By entering your email below, you confirm that this is your information.",
 	"Enter your email address": "Enter your email address",
 	"NEXT": "NEXT",
-	"Please enter a valid email address": "Please enter a valid email address",
+	"Please enter a valid email address": "Please enter a valid email address.",
```
Comment (lionell-pack-ttd) [inline public/locales_euid/en.json:10]: I think this key requires a '.' at the end now.

File: views_uid2/email_verified.hbs [hunk 2]
```diff
@@ -1,291 +1,112 @@
...
-      function cancelOptout() {
-          document.getElementById("prompt-modal").style.display = "none";
-      }
-      function promptOptout() {
-          document.getElementById("prompt-modal").style.display = "block";
-      }
+    function submitOptout() { document.getElementById("optout-form").submit(); }
+    function cancelOptout() {
+    document.getElementById("prompt-modal").style.display = "none"; } function
+    promptOptout() { document.getElementById("prompt-modal").style.display =
```
Comment (lionell-pack-ttd) [inline views_uid2/email_verified.hbs:108]: It looks like something's gone wrong with the formatting here.

**Comment by tynewton** [comment]
> Mostly it looks great - but the privacy page on EUID seems to be broken. image
> 
> I took a quick look and switching `height: 100vh` on `.main` in `style.css` to `min-height: 100vh` seems to fix it without breaking other things (looks like the height setting was being used to make the vertical positioning work on smaller pages).

This is resolved now.

**Pull request merged**